### DISCUSSION TITLE
About firmware

### DISCUSSION THREAD

**Discussion by sunxd001** [gusmanb/logicanalyzer#159] [Q&A]
I have read the open source firmware,in the function startcaprurefast,startcapturecomplex and startcapturesimple there is a parameter prelenth,I think that is capture length before trigger,but there is no use in these function.thank you for you reply!

**Comment by gusmanb** [comment]
Do you mean "PreSamples"?

It is used.

![imagen](https://github.com/user-attachments/assets/13acadd6-912c-40d3-b974-a017aad520e5)
